src/typeMapper.js

```diff
@@ -1,12 +1,22 @@
 import {
-  GraphQLInt,
-  GraphQLString,
+   GraphQLInt,
+   GraphQLString,
    GraphQLBoolean,
    GraphQLFloat,
    GraphQLEnumType,
    GraphQLList
  } from 'graphql';
 
+let _userMappingFunc;
+/**
+ * A function to set a custom mapping of types
+ * @param mapFunc
+ */
+export const mapType = (mapFunc = () => false) => {
+    _userMappingFunc = mapFunc;
+};
```

Comment:
Don't really need a default for the argument do we, if the user calls it they should really have a method :p